docs/csi-ephemeral-volume.md

```diff
@@ -0,0 +1,25 @@
+# CSI Ephemeral Volume
+
+## Introduction
+Traditionally, CSI volumes are used with a PersistentVolume and PersistentVolumeClaim object combination, whose lifecycle is independent of Pods. The ephemeral volume feature allows CSI volumes to be specified directly in the pod specification.  The lifecycle of ephemeral volumes is bound to the pod, being created when pod is created and deleted when pod is terminated.
```

Comment:
being created when the pod ...
and deleted when the pod ...

Reply:
@sdodsley resolved too